src/templates/person.html

```diff
@@ -562,6 +562,7 @@ <h3 id="account"> Account for {{ person.email }} </h3>
       <td>{{ identification.reference }}</td>
       <td>
         <form class="form-inline autosubmitonchange" method="POST" action="/__BACKOFFICE__/setIdentificationState/{{ person.email }}?method={{ identification.method }}">
+          <input type="hidden" name="skipSettingScreeningValues"value="true" />
```

Comment:
Why is this input `hidden`?
Is it because the value is set from the BE side?

Reply:
```suggestion
          <input type="hidden" name="skipSettingScreeningValues" value="true" />
```

Reply:
@UgurGumushan https://github.com/kontist/mock-solaris/pull/66#discussion_r923181682 🙂

Reply:
On manual tests, screening values can be set from the control panel. 
Some e2e tests depend on setting these values and expecting "green" screening values to be set. Because those tests won't pass this value the previous behavior would be kept.